src/sections/Projects/Sistent/components/card/index.js

```diff
@@ -0,0 +1,198 @@
+                </CardContent>
+                <CardActions>
+                  <Button size="small" variant="outlined">
+                    Learn More
+                  </Button>
+                </CardActions>
+              </Card>
+            </SistentThemeProvider>
+          </Row>
+
+          <h3>Main Action</h3>
+          <p>
+            1. A card often lets users interact with its entire surface to
+            trigger its main function, whether it&apos;s expanding, navigating
+            to another screen, or activating some other behavior.
+          </p>
+          <Row $Hcenter className="image-container">
+            <SistentThemeProvider initialMode={isDark ? "dark" : "light"}>
+              <Card sx={{ maxWidth: 345 }} variant="outlined">
+                <CardActions>
+                  <CardMedia
+                    component="img"
+                    height="80"
+                    image={MesheryLogo}
+                    alt="Meshery"
+                  />
+                  <CardContent>
+                    <Typography gutterBottom variant="h5" component="div">
+                      Meshery
+                    </Typography>
+                    <Typography variant="body" sx={{ color: "#9d9d9d" }}>
+                      Meshery is an open-source cloud-native management platform
+                      designed to help users manage, monitor, and optimize the
+                      performance of service meshes and their workloads.
+                    </Typography>
+                  </CardContent>
+                </CardActions>
+              </Card>
+            </SistentThemeProvider>
+          </Row>
+
+          <p>
+            2. Cards can also include secondary actions, which should be
+            positioned away from the primary action area to avoid unintentional
+            interaction conflicts.
+          </p>
+          <Row $Hcenter className="image-container">
+            <SistentThemeProvider initialMode={isDark ? "dark" : "light"}>
+              <Card sx={{ maxWidth: 345 }} variant="outlined">
+                <CardActions>
+                  <CardMedia
+                    component="img"
+                    height="90"
+                    image={KanvasLogo}
+                    alt="Kanvas"
+                  />
+                  <CardContent>
+                    <Typography gutterBottom variant="h5" component="div">
+                      Kanvas
+                    </Typography>
+                    <Typography variant="body" sx={{ color: "#9d9d9d" }}>
+                      Kanvas is a collaborative design tool that allows users to
+                      create, edit, and manage visual assets seamlessly within a
+                      shared workspace.
+                    </Typography>
+                  </CardContent>
+                </CardActions>
+                <CardActions>
+                  <Button size="small" color="primary" variant="outlined">
```

Comment:
```suggestion
                  <Button size="small" color="primary" variant="outlined">
```
generally we don't use mui's color prop you can make it default color rather that specifying color as primary
<img width="642" alt="Screenshot 2025-01-23 at 2 28 52 AM" src="https://github.com/user-attachments/assets/1eadaf3e-fe4c-41e4-81f4-087eb5ebf9b8" />